### PR TITLE
DMARC not consistently denoted by failures

### PR DESCRIPTION
**Pull request by AdamPrins**
Most locations marked what control failed,
But the DMARC graphs marked which control passed

All locations now go by the failure to be consistent.

### CODE REVIEW

**Review by FestiveKyle — CHANGES_REQUESTED** [review]
Translations need to be updated in the locales/*.po files. Default translation entries can be generated in these files by running `npm run extract` as long as the text to be translated exists inside either a \<Trans /> tag or t``

File: frontend/src/DmarcReportPage.js [hunk 1]
```diff
@@ -211,14 +211,14 @@ export default function DmarcReportPage({ summaryListResponsiveWidth }) {
...
+          name: 'failDkim',
+          displayName: t`Fail DKIM`,
         },
       ],
       moderateAlt: [
         {
-          name: 'passDkimOnly',
-          displayName: t`Pass Only DKIM`,
+          name: 'failSpf',
+          displayName: t`Fail SPF`,
```
Comment (FestiveKyle) [inline frontend/src/DmarcReportPage.js:221]: These fields need their 'name' properties to stay 'passSpfOnly' and 'passDkimOnly'. These names come from the API. The display names are good though!

**Review by FestiveKyle — CHANGES_REQUESTED** [review]

**Comment by FestiveKyle** [comment]
I fixed the merge conflict here. The locales make for some annoying conflicts sometimes, it wasn't bad this time though

**Comment by AdamPrins** [comment]
Thanks, I appreciate it.
Good to merge after the tests then?

**Review by FestiveKyle — APPROVED** [review]
👍

**Pull request merged**